lib/rate_limit/worker.rb

```
@@ -8,7 +8,7 @@ class Worker
 
     def initialize(topic:, value:)
       @topic     = topic.to_s
-      @value     = value.to_i
+      @value     = value.to_s
```

nitpick: perhaps we want to raise an error specific to `value should be a string or convertible object` ?

---

Good point 👍  I'll create an issue for general param validations